src/zepben/evolve/model/busbranch/__init__.py

```diff
@@ -0,0 +1,7 @@
+#  Copyright 2020 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from zepben.evolve.model.busbranch import *
```

Comment:
this will require users to do
```from zepben.evolve.model.busbranch import create_bus_branch_model```
is that what we want, or do we want this exposed under zepben.evolve?

I'm fine with it for the moment seeing as it is all in flux anyway, but something to think about.